pyxem/signals/diffraction2d.py

```diff
@@ -2091,7 +2117,7 @@ def get_azimuthal_integral2d(
         radial_range=None,
         azimuth_range=None,
         inplace=False,
-        method="splitpixel",
+        method="splitpixel_pyxem",
```

Comment:
I think we want to have multiple pyxem based methods eventually.  This might change over time as well.  We could have a `pyxem = True` argument as well if that makes more sense.